src/box/lua/config/source/env.lua

```diff
@@ -20,20 +26,23 @@ local function sync(_config_module, _iconfig)
         assert(latest_version ~= nil)
         instance_config:set(values, 'config.version', latest_version)
     end
+
+    self.values = values
+end
+
+-- Access the configuration after source:sync().
+function methods.get(self)
+    return self.values
 end
 
-local function get()
-    return values
+local function new()
+    return setmetatable({
+        name = 'env',
+        type = 'instance',
+        values = {},
```

Comment:
I would name it `_values` to highlight that this is considered as a private field and to encourage the outer code to call `:get()`. The same for the file source.

Reply:
Done for the both sources. Done for the etcd source in the EE part.